intro-HPC/ch_easybuild.tex

```diff
@@ -0,0 +1,139 @@
+\chapter{Easybuild}
+\label{ch:easybuild}
+
+\section{What is Easybuild?}
+
+You can use EasyBuild to build and install supported software in your own VSC account,
+rather than requesting a central installation by the HPC support team.
+
+EasyBuild is the software build and installation framework that was created by the HPC-UGent
+team, and has recently been picked up by HPC sites around the world. It allows you to manage
+(scientific) software on High Performance Computing (HPC) systems in an efficient way.
+
+\section{When should I use Easybuild?}
+
+For general software installation requests, please see \autoref{sec:software-installation}. However,
+there might be reasons to install the software yourself:
+
+\begin{enumerate}
+    \item applying custom patches to the software that only you or your group are using
+    \item evaluating new software versions prior to requesting a central software installation
+    \item installing (very) old software versions that are no longer eligible for central installation (on new systems)
+\end{enumerate}
+
+\section{Configuring EasyBuild}
+
+Before you use EasyBuild, you need to configure it:
+
+\subsection{Path to sources}
+
+This is where EasyBuild can find software sources:
+
+\begin{prompt}
+%\shellcmd{export EASYBUILD\_SOURCEPATH=/apps/gent/source}%
```

Comment:
Actually, this is wrong, since EasyBuild will use the first directory listed here to download new source tarballs to, and it can't because a user doesn't have write access to `/apps/gent/source`.

Please use:

```
export EASYBUILD\_SOURCEPATH=$VSC_DATA/easybuild/sources:/apps/gent/source
```

And explain:

* `$VSC_DATA/easybuild/sources` is where EasyBuild will (try to) automatically download sources if they're not available yet
* `/apps/gent/source` is the central "cache" for already downloaded sources, and will be considered by EasyBuild before downloading anything